App.tsx

```diff
@@ -7,8 +7,11 @@
 // export default App;
 
 import 'react-native-gesture-handler';
+import EStyleSheet from 'react-native-extended-stylesheet';
```

Comment:
What is the exact requirement of this package in this component?

Reply:
It's used to call the build method, without which EStyleSheet won't work.

EStyleSheet.build({});